CONTRIBUTING.md

```diff
@@ -92,10 +103,23 @@ Look for issues without the label `assigned` on it. This indicates that someone
 To start, comment on the issue asking if you can work on it. A maintainer will tag your username to the issue and apply the `assigned` label on it.
 
 ## Setting up a Development Environment
+
 For hassle-free code contribution, follow the steps below. NOTE: I mostly work in a Linux environment using a basic text editor (vim). Your set-up could be different. If it is, please feel free to submit an issue regarding your development environment and a pull request to add to this documentation.
 
+## Visual Studio Code Development Container
+
+Developing using Visual Studio Code development container lets you use a Docker container as a development environment. The Tern repository contains a definition for a pre-configured Docker container to simplify the setup process. To use it you need:
+
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Docker](https://www.docker.com/get-started)
+- [The Visual Studio Code Remote - Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+
+Opening this repository using Visual Studio Code will prompt you to re-open the workspace in a remote container. If not, you can press Ctrl+Shift+P to open the command palette and search for "Remote-Containers: Rebuild and Reopen in Container"
```

Comment:
`Remote-Containers: Rebuild and Reopen in Container` was not a valid search option for me. 
![Screen Shot 2021-03-17 at 7 26 29 PM](https://user-images.githubusercontent.com/12040351/111565438-95800900-8758-11eb-8143-d4d2159a080d.png)

Reply:
Maybe "Rebuild and Reopen in Container" was an option for me because I previously opened the devcontainer? Was "Open in Container" an option for you?

Reply:
"Open Folder in Container" is what I used and then I just selected my already cloned tern fork

Reply:
There was also "Clone repository in Container Volume" which may have worked but I didn't try that.